action.yml

```
@@ -8,6 +8,8 @@ inputs:
     description: 'The GITHUB_TOKEN secret'
   custom-keywords:
     description: 'List of custom keywords which should be included in the parser'
+  from-title:
+    description: 'To parse the issue numbers from title, a boolean'
```

```suggestion
  from-title:
    description: 'To parse the issue numbers from title, a boolean'
    required: false
```

This parameter should be optional

---

Nice catch..